cunumeric/module.py

```diff
@@ -1138,6 +1138,125 @@ def moveaxis(
 # Changing number of dimensions
 
 
+def _reshape_recur(ndim: int, arr: ndarray) -> tuple[int]:
+    if arr.ndim < ndim:
+        cur_shape = _reshape_recur(ndim - 1, arr)
+        if ndim == 2:
+            cur_shape = (1,) + cur_shape
+        else:
+            cur_shape = cur_shape + (1,)
+    else:
+        cur_shape = arr.shape
+    return cur_shape
+
+
+def _atleast_nd(
+    ndim: int, arys: Sequence[ndarray], view: Optional[bool] = True
+) -> Union(list[ndarray], ndarray):
+    inputs = arys = list(convert_to_cunumeric_ndarray(arr) for arr in arys)
+    if view:
+        inputs = list(arr.view() for arr in arys)
+    # 'reshape' change the shape of arrays
+    # only when arr.shape != _reshape_recur(ndim,arr)
+    result = list(arr.reshape(_reshape_recur(ndim, arr)) for arr in inputs)
+    # if the number of arrys in `arys` is 1, the return value is a single array
+    if len(result) == 1:
+        result = result[0]
+    return result
+
+
+def atleast_1d(*arys: Sequence[ndarray]) -> Union(list[ndarray], ndarray):
+    """
+
+    Convert inputs to arrays with at least one dimension.
+    Scalar inputs are converted to 1-dimensional arrays,
+    whilst higher-dimensional inputs are preserved.
+
+    Parameters
+    ----------
+    arys1, arys2, … : array_like
```

Comment:
Use this instead:

```
    *arys : array_like
```

Otherwise sphinx treats this as 3 separate arguments.

Similarly for `atleast_2d` and `atleast_3d`.